launch.py

```diff
@@ -12,21 +12,25 @@
 parser.add_argument('-l', '--label', default='INBOX')
 parser.add_argument('-p', '--prefix', default='\uf0e0')
 parser.add_argument('-c', '--color', default='#e06c75')
+parser.add_argument('-t', '--text-color', default='#ffffff')
 parser.add_argument('-ns', '--nosound', action='store_true')
 args = parser.parse_args()
 
 DIR = Path(__file__).resolve().parent
+PREV_COUNT_DIR = '/dev/shm/mail_count'
```

Comment:
As I know, `/dev/shm` is for passing data between programs (IPC). Better use just `/tmp` here. Explanation is here https://superuser.com/a/45509/1157520

Reply:
good idea!